var/spack/repos/builtin/packages/hoppet/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install hoppet
+#
+# You can edit this file again by typing:
+#
+#     spack edit hoppet
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Hoppet(AutotoolsPackage):
+    """A Fortran 95 package for carrying out QCD DGLAP evolution and other common manipulations of parton distribution functions (PDFs)."""
+
+    homepage = "https://hoppet.hepforge.org/"
+    url      = "https://github.com/gavinsalam/hoppet/archive/refs/tags/hoppet-1.2.0.tar.gz"
+
+
+    # maintainers = ['haralmha']
```

Comment:
If you are ok, with being pinged for reviews for this recipe, uncomment, if not, remove this line